### DISCUSSION TITLE
Develop to be frontend-only project or full-stack of monorepo ?

### DISCUSSION THREAD

**Discussion by zhou-tao** [zhou-tao/vue-power-admin#5] [General]
You want to frontend-only project or full-stack of monorepo ?

1. pure front-end

2. full-stack monorepo (nodejs)

**Comment by Blackman99** [comment]
According to my experience, it is rare that backend uses NodeJS. 
NodeJS is normally used as a middle integration layer instead of directly docking with DB.
It might be a good way to leave the DB docking to developers and provide a more lightweight in-memory store strategy.

**Reply by zhou-tao** [reply]
yeah, it's a good suggestion.
Do you have some good recommendations for a lightweight in-memory store strategy?

**Reply by Blackman99** [reply]
You can take a look at PocketBase: https://pocketbase.io/docs/
